### ISSUE TITLE
sickRage does not find episodes available on torrents

### ISSUE THREAD

**Issue by ofirda** [SiCKRAGETV/sickrage-issues#691]
Some episodes being search on the pre configured torrents cannot be found (even though exists in torrent website)

Example - Banshee season 3 (any episode) here's the log 

```
2015-02-17 00:47:41 INFO SEARCHQUEUE-MANUAL-259765 :: Unable to find a download for: [Banshee - 3x06 - We Were All Someone Else Yesterday] 2015-02-17 00:47:39 INFO SEARCHQUEUE-MANUAL-259765 :: [ThePirateBay] :: Performing episode search for Banshee 2015-02-17 00:47:39 INFO SEARCHQUEUE-MANUAL-259765 :: [EZRSS] :: Performing episode search for Banshee 2015-02-17 00:47:39 INFO SEARCHQUEUE-MANUAL-259765 :: [OldPirateBay] :: Performing episode search for Banshee 2015-02-17 00:47:38 INFO SEARCHQUEUE-MANUAL-259765 :: Performing season pack search for Banshee 2015-02-17 00:47:38 INFO SEARCHQUEUE-MANUAL-259765 :: FALLBACK SEASON PACK SEARCH INITIATED ... 2015-02-17 00:47:37 INFO SEARCHQUEUE-MANUAL-259765 :: [KickAssTorrents] :: Performing episode search for Banshee 2015-02-17 00:47:37 INFO SEARCHQUEUE-MANUAL-259765 :: Building internal name cache for Banshee 2015-02-17 00:47:37 INFO SEARCHQUEUE-MANUAL-259765 :: No scene exceptions update needed 2015-02-17 00:47:37 INFO SEARCHQUEUE-MANUAL-259765 :: Beginning manual search for: [Banshee - 3x06 - We Were All Someone Else Yesterday] 2015-02-17 00:47:29 INFO SEARCHQUEUE-MANUAL-259765 :: Unable to find a download for: [Banshee - 3x01 - The Fire Trials] 2015-02-17 00:47:28 INFO SEARCHQUEUE-MANUAL-259765 :: Performing season pack search for Banshee 2015-02-17 00:47:28 INFO SEARCHQUEUE-MANUAL-259765 :: FALLBACK SEASON PACK SEARCH INITIATED ... 2015-02-17 00:47:28 INFO SEARCHQUEUE-MANUAL-259765 :: [KickAssTorrents] :: Performing episode search for Banshee 2015-02-17 00:47:27 INFO SEARCHQUEUE-MANUAL-259765 :: [OldPirateBay] :: Performing episode search for Banshee 2015-02-17 00:47:27 INFO SEARCHQUEUE-MANUAL-259765 :: [EZRSS] :: Performing episode search for Banshee 2015-02-17 00:47:26 INFO SEARCHQUEUE-MANUAL-259765 :: [ThePirateBay] :: Performing episode search for Banshee 2015-02-17 00:47:16 INFO SEARCHQUEUE-MANUAL-259765 :: Building internal name cache for Banshee 2015-02-17 00:47:16 INFO SEARCHQUEUE-MANUAL-259765 :: No scene exceptions update needed 2015-02-17 00:47:16 INFO SEARCHQUEUE-MANUAL-259765 :: Beginning manual search for: [Banshee - 3x01 - The Fire Trials] I
```
This is easily found on the oldPiarteBay or kickass.to but not picked up by SR

**Comment by fernandog** [comment]
@ofirda we need the FULL log

**Comment by vdhmaaarten** [comment]
http://www.reddit.com/r/sickbeard/comments/2vb18o/access_provider_via_proxy_in_sickrage_is_not/

kickass.to is taken offline

**Comment by fernandog** [comment]
Please update SR.
TPB has an issue with season pack search 
Can you find the quality you want using browser

**Comment by ofirda** [comment]
Hi

I am out on a business trip

Please keep issue as open and I will attach full log once I am back

Thanks

On Saturday, February 28, 2015, Fernando <notifications@github.com> wrote:

> Closed #691 <https://github.com/SiCKRAGETV/sickrage-issues/issues/691>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SiCKRAGETV/sickrage-issues/issues/691#event-243753697>
> .
>


-- 
Best,
OfirD
